### PR TITLE
[M1606] Benthic Photo Quadrat ingest

### PR DESCRIPTION
**Pull request by gridcell**
- Includes serializers and tests

### CODE REVIEW

**Comment by gridcell** [comment]
[photo quadrat collection ingestion](https://trello.com/c/YjAERCZu/1606-photo-quadrat-collection-ingestion)

File: src/api/ingest/benthicpqt_serializer.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+def growth_form_choices():
+    return build_choices(GrowthForm.objects.choices(order_by="name"))
+
+
+def reef_slopes_choices():
+    return build_choices(ReefSlope.objects.choices(order_by="name"))
+
+
+def benthic_attributes_choices():
+    return [
```
Comment (saanobhaai) [inline src/api/ingest/benthicpqt_serializer.py:26]: Can't this use `build_choices` too?

File: src/api/resources/collect_record.py [hunk 2]
```diff
@@ -12,7 +12,16 @@
...
 from .mixins import CreateOrUpdateSerializerMixin
 from ..ingest.utils import InvalidSchema, ingest
-from ..models import CollectRecord
+from ..models import (
+    BENTHICLIT_PROTOCOL,
+    BENTHICPIT_PROTOCOL,
+    FISHBELT_PROTOCOL,
+    HABITATCOMPLEXITY_PROTOCOL,
+    BLEACHINGQC_PROTOCOL,
+    BENTHIC_PHOTO_QUADRAT_TRANSECT,
```
Comment (saanobhaai) [inline src/api/resources/collect_record.py:21]: I don't see where the `PROTOCOL` imports, esp. not the BQT one, are used

File: src/api/submission/utils.py [hunk 3]
```diff
@@ -8,7 +8,17 @@
...
 
 from ..mocks import MockRequest
-from ..models import AuditRecord, CollectRecord
+from ..models import (
+    BENTHICLIT_PROTOCOL,
+    BENTHICPIT_PROTOCOL,
+    FISHBELT_PROTOCOL,
+    HABITATCOMPLEXITY_PROTOCOL,
+    BLEACHINGQC_PROTOCOL,
+    BENTHIC_PHOTO_QUADRAT_TRANSECT,
```
Comment (saanobhaai) [inline src/api/submission/utils.py:17]: Don't we need to add the new BQT protocol to the `elif` blocks below?

**Review by saanobhaai — CHANGES_REQUESTED** [review]
Awesome. Couple of import questions.

File: src/api/submission/utils.py (hunk 3 above)
Comment (gridcell) [reply src/api/submission/utils.py:17]: We will but that'll be another ticket for validation/submission

File: src/api/ingest/benthicpqt_serializer.py (hunk 1 above)
Comment (gridcell) [reply src/api/ingest/benthicpqt_serializer.py:26]: build_choices expects the choices to be a dictionary not a model instance

**Pull request merged**